content/blog/bare-metal-provisioning-on-hpe-greenlake-using-terraform.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Bare Metal provisioning on HPE GreenLake using Terraform
+date: 2023-02-26T19:38:34.667Z
+author: Chaita Mylarappachar
+authorimage: /img/chaitra-mylarappachar-192.png
+disable: false
+tags:
+  - hpe-greenlake
+  - terraform
+  - bare-metal-as-a-service
+  - hpegl
+  - BMaaS
+---
+# Introduction
+
+The HPE GreenLake for Private Cloud Enterprise: bare metal (BMaaS) service offering enables you to create dedicated compute instances deployed on a physical IT infrastructure facilitating on-demand scalability, convenience, and agility as a cloud service. 
+Using BMaaS, you can create compute-instances provisioned with specific operating systems, network connections, one or more public SSH keys, and optional network-attached storage volumes.
```

Comment:
In the preview mode, lines 16 and 17 combine into one large paragraph. For ease of readability, you may wish to place a carriage return between the two so this would start the second paragraph.